src/TimeoutMigrationTool.Asp.IntegrationTests/AspTimeoutsSourceTests.cs

```diff
@@ -0,0 +1,473 @@
+﻿namespace TimeoutMigrationTool.Asp.IntegrationTests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Globalization;
+    using System.IO;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+    using Azure.Storage.Blobs;
+    using Microsoft.Azure.Cosmos.Table;
+    using Newtonsoft.Json;
+    using NUnit.Framework;
+    using Particular.TimeoutMigrationTool;
+    using Particular.TimeoutMigrationTool.Asp;
+
+    [TestFixture]
+    public class AspTimeoutsSourceTests
+    {
+        static readonly string connectionString = Environment.GetEnvironmentVariable("AzureStorageQueue_ConnectionString") ?? "UseDevelopmentStorage=true;";
```

Comment:
`AzureStorageQueue_ConnectionString`

Since we use this value for ASP and ASQ, should we rename it on TC to be: `AzureStorage_ConnectionString`?